### PR TITLE
Implement options related methods in State

### PR DESCRIPTION
**Pull request by Hazurl**

### CODE REVIEW

**Comment by Hazurl** [comment]
[Implement option related methods on State](https://trello.com/c/UE1dFLth/25-implement-option-related-methods-on-state)

File: src/state.py [hunk 1]
```diff
@@ -58,12 +65,12 @@ def is_end(self):
 
     def get_option_count(self):
         """Return the nuber of option in this state"""
-        pass
+        return self.options.count()
```
Comment (Jackojc) [inline src/state.py]: do we really need this?

Comment (Hazurl) [reply src/state.py]: yes, unless we replace `get_option` and `get_option_count` by a method `get_options` that return the list of options.

Comment (Jackojc) [reply src/state.py]: get_options would be nice.

**Pull request merged**